src/software/guides/addin-storage.md

```diff
@@ -0,0 +1,384 @@
+
+**customer.name**
+
+```javascript
+api.call("Get",
+{
+  "typeName": "AddInData",
+  "search": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "selectClause": "customer.name"
+  }
+});
+```
+
+The returned AddInData object will contain a value of "joesmith" in itsdata property.
+
+If you have an array in the path, it must be indicated by a [] after the name of the array property.
+
+For example, if you wanted to modify Example 4 to select all item names, we would add the following to the SELECT clause of the AddInDataSearch object:
+
+**items.[].name**
+
+```javascript
+api.call("Get",
+{
+  "typeName": "AddInData",
+  "search": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "selectClause": "items.[].name"
+  }
+});
+```
+
+The same notation is used for theWHERE clause. This notation can be used to drill down to as many objects as you want.
+
+### **Operators and Arguments**
+
+The WHERE clause of the AddInDataSearch object supports the following operators:
+
+| = | Equal to |
+| --- | --- |
+| \&lt; | Less than |
+| \&gt; | Greater than |
+| \&lt;= | Less than or equal to |
+| \&gt;= | Greater than or equal to |
+
+These can be used with the object path notation explained above.
+
+For example, if you want to get all items with a price less than 20, the appropriate WHERE clause will be:
+
+**items.[].price \&lt; 20**
+
+**Note** : The data type of the value on the right side of the operator is important. String values will need to be enclosed in quotation marks and properly escaped.
+
+To get all customers with the name "joesmith", the appropriate WHERE clause will be:
+
+**customer.name = "joesmith"**
+
+### **Important Operation Notes for Using the Get.AddInData**
+
+- The SELECT clause must be included if the WHERE clause is specified, otherwise the entire data object will be returned.
+- The GET operation always returnsan Array of AddInData objects, each with a unique value in the data property.
+- Search matching is case sensitive. In the examples above, searching for customer.name = "JoeSmith" will not return any results
+- Results returned by the SELECT and WHERE clauses will be in the scope of the entire AddInData object.To have a search return separate matches, the independent pieces of content must be added to separate AddInData objects using the ADD operation.
+
+## Updating Stored AddInData Content
+
+To update stored content, use the SET method on a AddInData object while specifying its GUID and ID. The return value is always null.
+
+As an example, use the [API Runner tool](https://my.geotab.com/sdk/api/apiRunner.html) to perform the following operation:
+
+```javascript
+api.call("Set",
+{
+  "typeName": "AddInData",
+  "entity": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "a6tClQu4iFkuroNPqOQsydg",
+    "groups": [{
+        "id": "GroupCompanyId"
+    }],
+    "details": JSON.stringify({
```

Comment:
Stringify not required on `details`

Reply:
Fixed in 764501adf2514edddd8a2468e0a9123abd659036